### ISSUE TITLE
 Invalid operation: schema "pg_temp_17" does not exist;

### ISSUE THREAD

**Issue by gorros** [awslabs/amazon-redshift-utils#490]
I am getting 
```
[3F000][500310] [Amazon](500310) Invalid operation: schema "pg_temp_17" does not exist;
```
when I query `v_get_obj_priv_by_user` 
```
select * from admin.v_get_obj_priv_by_user limit 100;
```

**Comment by jdahlbom** [comment]
This error is still there, please comment on this awslabs.

**Comment by saeedma8** [comment]
This should be fixed with issue #638. Please check it and let us know if the issue still persists. Thank you.
